src/main/kotlin/model/WMT.kt

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.hmpps.architecture
+
+import com.structurizr.model.Model
+import com.structurizr.model.SoftwareSystem
+import com.structurizr.view.ViewSet
+
+class WMT private constructor() {
+  companion object : HMPPSSoftwareSystem {
+    lateinit var system: SoftwareSystem
+
+    override fun defineModelEntities(model: Model) {
+      system = model.addSoftwareSystem(
+        "Workload Management Tool",
+        "(WMT) Helps probation practitioners schedule their time based on service user risk"
```

Comment:
📝 This definition slightly changed to reflect that this tool is only used in the community. As a result, we can use the "probation practitioners" and "service user" terms.